src/server/db/migrations/20250225000001_enable_unaccent.ts

```diff
@@ -0,0 +1,11 @@
+import { type Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+    CREATE EXTENSION unaccent;
+
+    CREATE INDEX idx_ign_communes_unaccent ON ign_communes USING gin (unaccent(nom) gin_trgm_ops);
```

Comment:
This creates an index on the nom column of the ign_communes table.
	•	It applies the unaccent function to remove accents before indexing.
	•	It uses a GIN index with trigram operations (gin_trgm_ops), which improves search performance for text queries, especially LIKE, ILIKE, and full-text searches.

Un peu de doc car je ne savais pas ce que c'était
:-)